ieeeuoft/hackathon-template
# IEEE-88: Moved current team api view

## Overview

- Resolves #IEEE-145
- moved class CurrentTeamAPIView from views.py to api_views.py


## Unit Tests Created

- none


## Steps to QA

- go to http://localhost:800/api/event/teams/team/ or send the same get request through swagger
- make sure the api response is correct and there are no other problems with the api
